meinberlin/apps/votes/models.py

```diff
@@ -67,6 +72,44 @@ def project(self):
     def is_valid_for_item(self, item):
         return item.module == self.module
 
+    @staticmethod
+    def get_hash_for_token(token, module):
+        params = argon2.profiles.RFC_9106_LOW_MEMORY
+        # use module_slug as salt as we can't store per-token salts
+        # TODO: slugs can belong, check max length of salt?
+        token_hash = argon2.low_level.hash_secret(
+            secret=token.encode(),
+            salt=module.slug.encode(),
```

Comment:
this uses the module slug to add at least a bit more security, however, if the slug is changed it wouldn't work anymore. It's not editable from the djang-admin afaics, so it should only happen if we ever do it manually?